src/client/components/MainWindow/MainWindow.jsx

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link, Switch, Redirect,
+} from 'react-router-dom';
 import PropTypes from 'prop-types';
 import SearchBlock from '../SearchBlock';
 import SingleMovie from '../SingleMovie';
 import './style.scss';
 
-const MainWindow = ({ showPage }) => (
+const NoMatch = ({ location }) => (
```

Comment:
There should be no more than one component in a file. Please extract it as a separate component.

Reply:
fixed.